products/motorola.md

```diff
@@ -0,0 +1,429 @@
+---
+title: Motorola Mobility
+category: device
+tags: mobile-phone
+iconSlug: motorola
+permalink: /motorola
+alternate_urls:
+-   /motorolamobility
+-   /motorola-mobility
+releasePolicyLink: https://en-us.support.motorola.com/app/software-security-update
+releaseColumn: false
+releaseDateColumn: true
+activeSupportColumn: false
+eolColumn: Security Updates
+
+# This data is parsed from the HTML CDATA on the Motorola website:
+# https://en-us.support.motorola.com/app/software-security-update
+#
+# See this Gist for a parsing script:
+#     https://gist.github.com/ab/67c6343530c6b8e599f07cedccbcba3d
```

Comment:
With a bit of additional work (mainly making the script retrieving the JSON data without a manual intervention ?) this could be moved to https://github.com/endoflife-date/release-data/. With this automation we could at least be alerted when new models needs to be documented.

> If they change the website format, then this will be difficult to update, but at least I figure this is a ton of useful info for all their current product lines.

I agree, and now that the script has been written let's use it ;).

Reply:
Thanks, I can make the script work without manual intervention. But how were you envisioning this would work with the `release-data` repo? It looks like that automation only pulls the `releaseDate` and name.

Whereas this script currently also extracts `eol` and `link`. Should I add support for those fields to `release-data/latest.py`? Or did you have a plan for how that should work?

Reply:
> It looks like that automation only pulls the releaseDate and name.

Yes, for now the script would only alert us when new models appear on the page.

> Whereas this script currently also extracts eol and link. Should I add support for those fields to release-data/latest.py? Or did you have a plan for how that should work?

Not for now, that's too much work given `release-data` data structure is not yet adapted to store such information. But I am (slowly) working toward that goal (no ETA though).